src/System.CommandLine/Invocation/ModelBindingCommandHandler.cs

```diff
@@ -11,39 +11,39 @@ namespace System.CommandLine.Invocation
 {
     internal class ModelBindingCommandHandler : ICommandHandler
     {
-        private readonly Delegate _handlerDelegate;
-        private readonly object _invocationTarget;
-        private readonly ModelBinder _invocationTargetBinder;
-        private readonly MethodInfo _handlerMethodInfo;
+        private readonly Delegate? _handlerDelegate;
+        private readonly object? _invocationTarget;
+        private readonly ModelBinder? _invocationTargetBinder;
+        private readonly MethodInfo? _handlerMethodInfo;
```

Comment:
could this field be marked non-nullable? (same question for `_parameterDescriptors`)